integration/tests/tracer/schemas/debug_traceCall.json

```diff
@@ -81,9 +81,7 @@
                         "gas",
                         "gasCost",
                         "op",
-                        "pc",
-                        "returnData",
```

Comment:
why you remove it? it should be by specification. If tracer doesn't return these fields, some ethereum clients could fail

Reply:
it is removed only from required fields. It is still in schema